pkg/virt-launcher/virtwrap/device/hostdevice/generic/hostdev_test.go

```diff
@@ -91,7 +91,7 @@ var _ = Describe("Generic HostDevice", func() {
 			Model:  "vfio-pci",
 		}
 
-		Expect(hostDevices, err).To(Equal([]api.HostDevice{expectHostDevice0, expectHostDevice1}))
+		Expect(hostDevices).To(Equal([]api.HostDevice{expectHostDevice0, expectHostDevice1}))
```

Comment:
nit: how about inlining the call into the expectation ? 

```golang
Expect(generic.CreateHostDevicesFromPools(vmi.Spec.Domain.Devices.HostDevices, pciPool, mdevPool)).To(Equal([]api.HostDevice{expectHostDevice0, expectHostDevice1}))
```

Reply:
Done